src/main/kotlin/org/malachite/estella/services/OrganizationService.kt

```diff
@@ -39,17 +39,31 @@ class OrganizationService(
             .let { organizationRepository.save(it) }
     }
 
+    fun addTask(uuid: String, task: Task) =
+        getOrganization(uuid)
+            .let {
+                val updatedTasks = it.tasks.plus(task)
+                it.copy(tasks = updatedTasks)
+            }
+            .let { organizationRepository.save(it) }
+
+    fun deleteTask(uuid: String, taskId: Int) =
+        getOrganization(uuid)
+            .let {
+                val updatedTasks = it.tasks.filter { it.id != taskId }.toSet()
+                it.copy(tasks = updatedTasks)
```

Comment:
Wydaje mi się, że tu tylko tracimy połączenie, a nijak nie usuwamy tego oryginalnego taska. Więc po usunięciu taska nie usuniemy go, a tylko go odłączymy, pozostawiając go w bazie na zawsze, nie mogąc nic z nim zrobić.

Reply:
tak, ale ta funkcja jest wołana w TaskService i z tego co rozumiem to ma własnie za zadanie tylko usunąć powiązanie, w taskServisie dodatkowo jest też usuwanie samego taska. Zgadzam się, że nazwa jest troche myląca, proponuje zawrzeć w nazwie funkcji informację że chodzi o usunięcie samego powiązania taska z organizacją

Reply:
Znaczy chyba nie dajemy możliwości by jeden task był w więcej niż jednej organizacji więc na końcu wywoływałbym z taskService'u uswanie danego taska by mieć pewność że nie będzie on sobie leżał w przestrzeni

Reply:
ale to sie dzieje, tylko w TaskService:
![obraz](https://user-images.githubusercontent.com/56320160/139666411-ab4bb596-095a-4187-971a-d8dc08f61033.png)